README.md

```diff
@@ -1 +1,57 @@
-# Abdifatah-capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+This project is a capstone project for module one in Microverse and the page is about conference.
+
+## Built With
+
+- Major languages
+  Html and css
+- Frameworks
+  no frameworks is used
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link]()
+
```

Comment:
I can see that the project is deployed and working properly in the github pages. Unfortunately the link you provided is not working. Kindly fix it :hammer_and_wrench: